system/ee/ExpressionEngine/Controller/Logs/Cp.php

```diff
@@ -37,10 +37,11 @@ public function index()
         $this->base_url->path = 'logs/cp';
         ee()->view->cp_page_title = lang('view_cp_log');
 
-        $logs = ee('Model')->get('CpLog')->with('Site');
+        $sites = ee('Model')->get('Site');
+        $logs = ee('Model')->get('CpLog');
 
         if ($search = ee()->input->get_post('filter_by_keyword')) {
-            $logs->search(['action', 'username', 'ip_address', 'Site.site_label'], $search);
```

Comment:
IMO there's no benefit to keyword searching `Site.site_label` because it's already a separate dropdown field with MSM enabled:

<img width="301" alt="image" src="https://github.com/user-attachments/assets/8b3c6178-dfa9-4d81-8b5a-31d737d1b778" />

And with MSM disabled, there's no benefit to searching on site_label because all results would be connected to the same site_id.